tests/acceptance/apps_dns/apps_dns_test.go

```diff
@@ -0,0 +1,309 @@
+/*
+Copyright 2020 SUSE
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package apps_dns_test
+
+import (
+	"fmt"
+	"math/rand"
+	"net"
+	"net/http"
+	"sort"
+
+	"github.com/miekg/dns"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/SUSE/kubecf-apps-dns/tests/acceptance/apps_dns/fake"
+)
+
+var _ = Describe("AppsDns", func() {
```

Comment:
Would it make sense to test results with a mix of IPv4 & IPv6? Or can we assume that the individual cases passing is enough?

Reply:
I believe it makes sense to mix them, yes. Technically, the plugin should deal well with this even it being something I don't believe the original service discovery controller implements.

Reply:
Added with 4604d34.